### ISSUE TITLE
configure inactive pane in colorscheme file

### ISSUE THREAD

**Issue by og900aero** [ranger/ranger#2847]
How configure inactive pane (when use dual pane function) for dimmed color. I found dim variables, but doesnt work.

```
from __future__ import (absolute_import, division, print_function)

from ranger.gui.colorscheme import ColorScheme
from ranger.gui.color import *

class custom(ColorScheme):
    progress_bar_color = 124

    def use(self, context):
        fg, bg, attr = default_colors

        if context.reset:
            return default_colors
            
        elif context.in_browser:
            if context.selected:
                fg = 0
                bg = 144
            else:
                fg = 144
                bg = 0
            **if context.inactive_pane:
                attr = dim**
            if context.border:
                fg = default
            if context.image:
                fg = 109
            if context.video:
                fg = 109
            if context.audio:
                fg = 109
            if context.document:
                fg = 73
            if context.container:
                fg = 61
            elif context.executable and not \
                    any((context.media, context.container,
                         context.fifo, context.socket, context.directory)):
                fg = 32
            if context.socket:
                fg = 136
            if context.fifo:
                fg = 136
            if context.device:
                fg = 244
            if context.link:
                fg = 37 if context.good else 160
                if context.bad:
                    bg = 9
```

**Comment by og900aero** [comment]
> if context.inactive_pane:
>                 attr = dim

Ohh. I try a lot since 2-3 days, and very strange. Have some directory, where the dim value doesnt work, but work in other directory and files. I dont understand why. The directoty permissions is the same, the conent mime type same too.

**Comment by og900aero** [comment]
Exactly what data do you need about the libraries involved?

Meanwhile, I tried two more things:

- The problem always occurs with the .Trash directory
- There are some directpry that don't have the problem if I try them when start ranger, but if I go to a subfolder after opening the ranger, then switch to the multipane view and go back to that main folder, the dim doesn't work anymore some directory. Example directory name: Android, Doki, Downloads, Edzések, Kepek, Guitar Video.

I use the following code to change view:

```
class toggle_view(Command):
    """:toggle_view

    """

    def execute(self):
        if len(self.fm.tabs) >= 2:
            self.fm.tab_move(-1)
            self.fm.tab_close()
        else:
            self.fm.tab_new('%d')
            self.fm.tab_move(-1)

map mm chain toggle_view; set viewmode!
```

**Comment by toonn** [comment]
A zip or a tarball containing a couple directories that show the problem.

Those names don't have much in common though so I'm really not sure why this would be happening. Do `^L` or `^R` fix the colors at all?

**Comment by toonn** [comment]
What are the errors? Anything related to colorschemes?

**Comment by og900aero** [comment]
> What are the errors? Anything related to colorschemes?

No. Attached screenshot, but no error in colorschemes, only /usr/local/lib/python3.9/dist-packages/ranger

[Picture_1](https://www.kepfeltoltes.eu/images/2023/05/23/426screenshot_1684820468.jpg)
[Picture_2](https://www.kepfeltoltes.eu/images/2023/05/23/358screenshot_1684820395.jpg)

**Comment by og900aero** [comment]
> The errors seem related to unbinding a key map potentially? Did you add any unmaps to `rc.conf`?

Yes, 132 pieces :) What I don't need, I put to unmap.

**Comment by toonn** [comment]
Could you comment out all but one unmap? I'd like to find out whether unmap is broken in general or just in specific cases.

**Comment by og900aero** [comment]
> Could you comment out all but one unmap? I'd like to find out whether unmap is broken in general or just in specific cases.

This causes error messages:

unmap sS
unmap sB
unmap sN
unmap sM
unmap sC
unmap sA
unmap sT
unmap sE

But that is my fault, because I don't see how such a key is used.

**Comment by og900aero** [comment]
I reinstalled my linux (i use own script), and now the problem is almost solved. The problem now only occurs with the .Trash directory.
